testsuite/MDAnalysisTests/test_auxiliary.py

```diff
@@ -35,7 +35,12 @@ def __init__(self):
 class BaseAuxReaderTest(object):
     def __init__(self, reference):
         self.ref = reference
-        self.reader = self.ref.reader('test', self.ref.testdata)
+
+    def setUp(self):
```

Comment:
I can't remember if you're allowed to use both `__init__` and `setUp`.... maybe stick to just using `setUp` and `tearDown` so it's all in one method